Updating wording for contacting support (#1045)
codatio/codat-docs: docs/get-started/support.md

draft: true
---

> Codat provides a second line support function during UK business hours (09:00 to 17:00 Monday to Friday) and can be contacted raising a ticket through [this form](https://codat.zendesk.com/hc/en-gb/requests/new).

Reviewer: ```suggestion
Codat provides a second line support function during UK business hours (09:00 to 17:00 Monday to Friday) and can be contacted via our [support request form](https://codat.zendesk.com/hc/en-gb/requests/new).
```

## Raising a ticket

> If you run into an issue and need to raise a support ticket, you can do so through [this form](https://codat.zendesk.com/hc/en-gb/requests/new).

Reviewer: ```suggestion
If you run into an issue and need to raise a support ticket, use our [support request form](https://codat.zendesk.com/hc/en-gb/requests/new).
```
